doc/ref_model/chapters/chapter07.rst

```diff
@@ -708,33 +709,91 @@ Access to the repository must be granted by a dedicated authorization. The code
++==============================+===================================================+
+| **Supplier Name**            | The name of an entity that creates, defines, and  |
+|                              | identifies components.                            |
++------------------------------+---------------------------------------------------+
+| **Component Name**           | Designation assigned to a unit of software        |
+|                              | defined by the original supplier.                 |
++------------------------------+---------------------------------------------------+
+| **Version of the Component** | Identifier used by the supplier to specify a      |
+|                              | change in software from a previously identified   |
+|                              | version.                                          |
++------------------------------+---------------------------------------------------+
+| **Other Unique Identifiers** | Other identifiers that are used to identify a     |
+|                              | component, or serve as a look-up key for relevant |
+|                              | databases.                                        |
++------------------------------+---------------------------------------------------+
+| **Dependency Relationship**  | Characterizing the relationship that an upstream  |
+|                              | component X is included in software Y.            |
++------------------------------+---------------------------------------------------+
+| **Author of SBOM Data**      | The name of the entity that creates the SBOM data |
+|                              | for this component.                               |
++------------------------------+---------------------------------------------------+
+| **Timestamp**                | Record of the date and time of the SBOM data      |
+|                              | assembly.                                         |
++------------------------------+---------------------------------------------------+
+**Table 7-2 **: SBOM Data Fields components, source `NTIA <https://www.ntia.doc.gov/files/ntia/publications/\
+sbom_minimum_elements_report.pdf>`_
+
+Refer to the document for more details on each data field. Examples of commonly used identifiers are provided.
+
+In order to use SBOMs efficiently and spread their adoption, information must be generated and shared in a standard 
+format. This format must be machine-readable to allow automation. Proprietary formats should not be used. 
+Multiple data formats exist covering baseline SBOM information. The three key formats, Software Package Data 
+eXchange (SPDX), CycloneDX, and Software Identification Tags (SWID tags) are interoperable for the core data fields 
+and use common data syntax representations.
+
+- `SPDX <https://spdx.dev/>`__ is an open-source machine-readable format developed under the umbrella of the Linux 
+  Foundation. 
+  
```

Comment:
can we remove the breaks to avoid confusion for the last paragraph in this sub-section,

Reply:
@karinesevilla No I was thinking of making each of SPDX, CycloneDX, etc. as bullets like in the following (with line breaks for line length limits):
- `SPDX <https://spdx.dev/>`__ is an open-source machine-readable format developed under the umbrella of the Linux Foundation.  The `SPDX specification 2.2 <https://spdx.dev/specifications/>`__ has been published as the standard  ISO/IEC 5962:2021. It provides a language for communicating the data, licenses, copyrights, and security information associated with software components. With the SPDX specification 2.2, multiple file formats are available: YAML,  JSON, RDF/XML, tag\:value flat text, and xls spreadsheets.
- CYcloneDX ...


It is OK if you want to leave as is.

Reply:
@pgoyal01 Okay I see, I will do it during proofreading